transformer/kubernetes/apiresource/argocdapplication.go

```diff
@@ -70,6 +71,16 @@ func (*ArgoCDApplication) createNewResource(irApplication irtypes.Application, t
 		clusterServer = deployToSameCluster
 	}
 	appGVK := v1alpha1.ApplicationSchemaGroupVersionKind
+	destNamespace := qaengine.FetchStringAnswer(
+		common.ConfigCICDArgoDestinationNameSpace,
+		"Enter the destination namespace for argo cd pipeline",
+		[]string{},
```

Comment:
Instead of providing empty context `[]string{}`, can we provide a context something like `[]string{If Argo CD pipeline is not relevant to you, then leave empty to use the default value for it.}`